chord_metadata_service/chord/workflows/wdls/experiments_json.wdl

```diff
@@ -25,3 +40,30 @@ task copy_task {
         File json_document = "ingest.json"
     }
 }
+
+task ingest_task {
+    input {
+        File json_document
+        String service_url
+        String dataset_id
+        String token
+    }
+    command <<<
+        RESPONSE=$(curl -X POST -k -s -w "%{http_code}" \
+            -H "Content-Type: application/json" \
+            -H "Authorization: ~{token}" \
+            --data "@~{json_document}" \
+            "~{service_url}/ingest/~{dataset_id}/experiments_json")
```

Comment:
replace service url with injected katsu url via the FROM_CONFIG value i've mentioned